src/utils/filterTodos.tsx

```diff
@@ -0,0 +1,15 @@
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (todos: Todo[], filterparam: string): Todo[] => {
+  let resultTodos = [...todos];
+
+  if (filterparam === 'active') {
```

Comment:
Also in this case you need to use global constants or enums ( watch comment above)

Reply:
I fix everything) But on add todo task step 3. Could u please approve this one?) thx